modules/openxr/openxr_interface.cpp

```diff
@@ -1541,6 +1562,22 @@ RID OpenXRInterface::get_vrs_texture() {
 	return xr_vrs.make_vrs_texture(target_size, eye_foci);
 }
 
+void OpenXRInterface::set_cpu_level(PerfSettingsLevel p_level) {
+	OpenXRPerformanceSettingsExtension::get_singleton()->set_cpu_level(p_level);
+}
+
+void OpenXRInterface::set_gpu_level(PerfSettingsLevel p_level) {
+	OpenXRPerformanceSettingsExtension::get_singleton()->set_gpu_level(p_level);
+}
```

Comment:
Shouldn't these methods check if the extension is available prior to invoking it?